backend/building-block/src/test/resources/config/application-mysql.yml

```diff
@@ -0,0 +1,16 @@
+spring:
+    datasource:
+        driver-class-name: com.mysql.cj.jdbc.Driver
+        url: jdbc:mysql://localhost:3363/building-block-test
```

Comment:
This is fine, but also add a comment to the docker-compose-base list so we don't use it twice in the future.